common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -289,7 +289,31 @@ def student_view(self, context):
                 masquerading_as_specific_student = context.get('specific_masquerade', False)
                 banner_text, special_html = special_html_view
                 if special_html and not masquerading_as_specific_student:
-                    return Fragment(special_html)
+                    fragment = Fragment(special_html)
+                    if 'You have submitted your timed exam.' in special_html:
```

Comment:
If a learner were to set their UI language to something other than English in their account profile, then this `if` block wouldn't trigger. (Not that we expect them to, but then there's nothing that keeps them from doing that either.) But I *think* you do get the same result if you simply replace this with

```python
if _('You have submitted your timed exam.') in special_html:
```

Reply:
Ack, I've made the change.

Reply:
I think you'll still want this, though :)

```python
from django.utils.translation import ugettext as _
```

No?

Reply:
this view has it defined as:
`_ = self.runtime.service(self, "i18n").ugettext`
I _think_ that should do the trick, right?

Reply:
Ah! Gotcha.